ValetTests/SynchronizableTests.swift

```diff
@@ -23,62 +23,59 @@ import Valet
 import XCTest
 
 
-// The iPhone simulator fakes entitlements, allowing us to test the iCloud Keychain (VALSynchronizableValet) code without writing a signed host app.
-#if (arch(i386) || arch(x86_64)) && os(iOS)
+// Until we can write a signed host app on macOS, we can only test the iCloud Keychain (SynchronizableValet) code on iOS.
+#if os(iOS)
```

Comment:
I'm a bit nervous about removal of the simulator `#if` logic here - e.g. I _think_ unit tests can be run on device and these will fail unless the build is signed